packages/js/src/initializers/term-scraper.js

```diff
@@ -416,13 +411,11 @@ export default function initTermScraper( $, store, editorData ) {
 		};
 
 		// Initialize the snippet editor data.
-		let snippetEditorData = getDataFromCollector( termScraper );
+
 		initializeCornerstoneContentAnalysis( app );
-		const snippetEditorTemplates = getTemplatesFromL10n( wpseoScriptData.metabox );
-		snippetEditorData = getDataWithTemplates( snippetEditorData, snippetEditorTemplates );
 
-		// Set the initial snippet editor data.
-		store.dispatch( updateData( snippetEditorData ) );
+		const snippetEditorData = getSnippetEditorData( store.getState() );
+		const snippetEditorTemplates = getTemplatesFromL10n( wpseoScriptData.metabox );
```

Comment:
The templates is only for SEO title & description, which you removed. So maybe also remove the templates? And just refer to slug.